physionet-django/events/views.py

```diff
@@ -114,53 +116,123 @@ def event_home(request):
             if event.host != user:
-                messages.error(request, "You don't have permission to accept/reject this application")
+                messages.error(
+                    request,
+                    "You don't have permission to accept/reject this application",
+                )
                 return redirect(event_home)
-            elif event_application.status == EventApplication.EventApplicationStatus.APPROVED:
-                if EventParticipant.objects.filter(event=event, user=event_application.user).exists():
+            elif (
+                event_application.status
+                == EventApplication.EventApplicationStatus.APPROVED
+            ):
+                if EventParticipant.objects.filter(
+                    event=event, user=event_application.user
+                ).exists():
                     messages.error(request, "Application was already approved")
                     return redirect(event_home)
                 event_application.accept(
-                    comment_to_applicant=form.cleaned_data.get('comment_to_applicant')
+                    comment_to_applicant=form.cleaned_data.get("comment_to_applicant")
                 )
                 notification.notify_participant_event_decision(
                     request=request,
                     user=event_application.user,
                     event=event_application.event,
                     decision=EventApplication.EventApplicationStatus.APPROVED.label,
-                    comment_to_applicant=form.cleaned_data.get('comment_to_applicant')
+                    comment_to_applicant=form.cleaned_data.get("comment_to_applicant"),
                 )
-            elif event_application.status == EventApplication.EventApplicationStatus.NOT_APPROVED:
+            elif (
+                event_application.status
+                == EventApplication.EventApplicationStatus.NOT_APPROVED
+            ):
                 event_application.reject(
-                    comment_to_applicant=form.cleaned_data.get('comment_to_applicant')
+                    comment_to_applicant=form.cleaned_data.get("comment_to_applicant")
                 )
                 notification.notify_participant_event_decision(
                     request=request,
                     user=event_application.user,
                     event=event_application.event,
                     decision=EventApplication.EventApplicationStatus.NOT_APPROVED.label,
-                    comment_to_applicant=form.cleaned_data.get('comment_to_applicant')
+                    comment_to_applicant=form.cleaned_data.get("comment_to_applicant"),
                 )
 
             return redirect(event_home)
         else:
             form_error = True
 
+    events = Event.objects.all().prefetch_related(
+        "participants",
+        "applications",
+    )
+
+    event_details = {}
+    for event in events:
+        all_applications = event.applications.all()
```

Comment:
Same logic as with `events_all` would apply here - wouldn't `applications` or `event_applications` be enough? It's just a thought though, I'm not a stickler for naming.

Reply:
No, definitely!
I think what Tom mentioned the other day in the meeting makes sense - it can always be looked up, and even though the naming looks more clear in the context of this PR, in terms of the whole codebase, it can look confusing. 
I've updated the naming here as well.